From a257c01e7ac7d17ee29bc7b8b568b0ca3ebea1b4 Mon Sep 17 00:00:00 2001
From: Ajay Singh Parmar <aparmar@codeaurora.org>
Date: Wed, 27 May 2015 14:07:39 -0700
Subject: [PATCH 1/2] msm8916: Add CEC support for 8916

Enable CEC HAL to integrate CEC framework and CEC hardware.

Change-Id: Iaa30d3f67beabf18d5a22e8a5a147b92889dca7b

Conflicts:
	system.prop
---
 dev_qcom_msm8916_nov_release.sh | 5 +++++
 msm8916.mk                      | 5 +++++
 system.prop                     | 3 +++
 3 files changed, 13 insertions(+)
 create mode 100644 dev_qcom_msm8916_nov_release.sh

diff --git a/dev_qcom_msm8916_nov_release.sh b/dev_qcom_msm8916_nov_release.sh
new file mode 100644
index 0000000..ba0c9dc
--- /dev/null
+++ b/dev_qcom_msm8916_nov_release.sh
@@ -0,0 +1,5 @@
+#CEC CHANGES
+git fetch ssh://review-android.quicinc.com:29418/platform/vendor/qcom/msm8916 refs/changes/46/1287646/3 && git cherry-pick FETCH_HEAD
+
+#ADSP CHANGES
+git fetch ssh://review-android.quicinc.com:29418/platform/vendor/qcom/msm8916 refs/changes/61/1425261/1 && git cherry-pick FETCH_HEAD
diff --git a/msm8916.mk b/msm8916.mk
index bc5775e..7b79d2c 100644
--- a/msm8916.mk
+++ b/msm8916.mk
@@ -94,3 +94,8 @@ PRODUCT_PACKAGES += \
     wpa_supplicant_overlay.conf \
     p2p_supplicant_overlay.conf
 
+#HDMI CEC
+PRODUCT_COPY_FILES += \
+    frameworks/native/data/etc/android.hardware.hdmi.cec.xml:system/etc/permissions/android.hardware
+.hdmi.cec.xml
+
diff --git a/system.prop b/system.prop
index a226e99..2112791 100644
--- a/system.prop
+++ b/system.prop
@@ -113,3 +113,6 @@ ro.sf.lcd_density=480
 
 # system property for PIP support
 persist.camera.pip.support=0
+#CEC settings
+ro.hdmi.device_type=4
+persist.sys.hdmi.addr.playback=4
-- 
1.8.2.1

